model/Software/Classes/Sbom.md

```diff
@@ -1,15 +1,18 @@
 SPDX-License-Identifier: Community-Spec-1.0
 
-# File
+# SBOM
```

Comment:
I thought we decided for `Sbom` as a class name

Reply:
Ah, the heading does not match the `name` of the class? Feels inconsistent and prone to introduce issues